### DISCUSSION TITLE
problem whit kde 6

### DISCUSSION THREAD

**Discussion by poncho-ajmv** [davatorium/rofi#1967] [(Possible) Issues]
After update i have this error 
**(process:16986): X11Helper-WARNING **: 08:13:46.325: Failed to grab keyboard after 5001 times. Giving up.**
Never before happend to me

**Comment by DaveDavenport** [comment]
I have no idea what kde6 does, don't use it myself.
It looks like rofi is not allowed to get exclusive rights to the keyboard.

Is KDE 6 X11 or Wayland?
